src/Elastic.CommonSchema/EcsDocument.Generated.cs

```diff
@@ -0,0 +1,223 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+/*
+IMPORTANT NOTE
+==============
+This file has been generated. 
+If you wish to submit a PR please modify the original csharp file and submit the PR with that change. Thanks!
+*/
+
+// ReSharper disable RedundantUsingDirective
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Net;
+using System.Runtime.Serialization;
+
+namespace Elastic.CommonSchema
+{
+	///<inheritdoc cref="BaseFieldSet"/>
+	public partial class EcsDocument : BaseFieldSet 
+	{
+
+		/// <summary>
+		/// Elastic Common Schema version v8.3.1
+		/// </summary>
+		public static string Version => "v8.3.1";
```

Comment:
Was that intended that before version had format "1.2.3" and now "v1.2.3"? Although [specification](https://www.elastic.co/guide/en/ecs/8.3/ecs-ecs.html) doesn't explicitly mention required format, example there uses "1.2.3" format.